app/controllers/api/v1/bookmarks_controller.rb

```diff
@@ -0,0 +1,117 @@
+class Api::V1::BookmarksController < Api::V1::BaseController
```

Comment:
Hmm, I was thinking about External Works, I realize now. And the actual bookmark object is the same, so all bookmark creation makes sense in BookmarksController, but where the process of creating external works belongs... IDK, probably it should be here after all and just made conditional on whether the url passed is for an AO3 work or not, that seems most parallel to the existing process since External Works controller just makes the new external work object and passes off the actual creation to the Bookmarks controller via the form.